README.md

```diff
@@ -1,2 +1,45 @@
-# new-relic
-Best practice New Relic configurations.
+# New Relic considerations
+
+Considerations for using New Relic in the federal government.
+
+Below is a description of the settings in your New Relic `.ini` file that concern privacy and security. Example configuration files can be found in this repository:
```

Comment:
We thought it would be something to consult. But making it an .ini would be fine with me too.